content/en/docs/reference/glossary/service-account.md

```diff
@@ -17,3 +17,10 @@ tags:
 
 When processes inside Pods access the cluster, they are authenticated by the API server as a particular service account, for example, `default`. When you create a Pod, if you do not specify a service account, it is automatically assigned the default service account in the same {{< glossary_tooltip text="Namespace" term_id="namespace" >}}.
 
+{{< comment >}}
+This is a hint for anyone localizing this page: you can *optionally* add
+a translated version of the following line:
+The API object representing a service account is: ServiceAccount.
+
+(if you translated “service account” into your localization's language, leave “ServiceAccount” verbatim)
+{{< /comment >}}
```

Comment:
There are many similar cases in the glossary such as PersistentVolume, PersistentVolumeClaim, PodSecurityPolicy, StorageClass. However, it is inconsistent to have the comment only in ServiceAccount. It is also not efficient to repeat the same comment for each object. Is there any other good place? How about adding it to `/content/docs/contribute/localization.md`?

Reply:
Mmm, that might work better.

Reply:
@sftim, What do you think about moving the comment to `contribute/localization.md`?
After that, I think the change is okay.
Squash your commits?

Reply:
I'll save https://github.com/kubernetes/website/pull/18044#discussion_r402681627 for a different PR.